Caculator/Caculator/ViewController.swift

```diff
@@ -9,11 +9,65 @@ import UIKit
 
 class ViewController: UIViewController {
 
+
+    @IBOutlet weak var zeroButton: UIButton!
```

Comment:
tương tự các IB dưới
```suggestion
    @IBOutlet private weak var zeroButton: UIButton!
```